src/parser/SeriesParser.ts

```diff
@@ -185,6 +185,7 @@ export class SeriesParser implements Parser<Series> {
         return {
           title: title,
           description: content[0].innerText,
+          htmlDescription: content[0].querySelector('div').innerHTML,
```

Comment:
Don't need to do another query selector, just do `content[0].innerHTML`. content is an array of elements which we got using a query selector :)